stat/distuv/laplace.go

```diff
@@ -192,14 +196,12 @@ func (l Laplace) Score(deriv []float64, x float64) []float64 {
 		deriv[0] = 1 / l.Scale
 	} else if diff < 0 {
 		deriv[0] = -1 / l.Scale
-	} else if diff == 0 {
-		deriv[0] = 0
 	} else {
 		// must be NaN
 		deriv[0] = math.NaN()
 	}
```

Comment:
Is this described as the convention somewhere?

Reply:
I didn't find anything like that. If the derivative doesn't exist, we should return NaN IMHO.